chapters/classes_and_objects/object-literal.md

```diff
@@ -20,7 +20,29 @@ window.MY_NAMESPACE ?= {}
 This is equivalent to the following JavaScript:
 
 {% highlight javascript %}
-window.MY_NAMESPACE = window.MY_NAMESPACE || {};
+if(window.MY_NAMESPACE == null) {
+  window.MY_NAMESPACE = window.MY_NAMESPACE
```

Comment:
Sure, sorry for noticing the typo, I did the changes quickly and that is never a good idea. 👍